python/tvm/topi/nn/sparse.py

```diff
@@ -207,3 +207,28 @@ def _csr_transpose_ir(data, indices, indptr, out_data, out_indices, out_indptr):
         last[0] = temp2[0]
 
     return irb.get()
+
+
+@tvm.target.generic_func
+def sparse_dense_alter_layout(_attrs, _inputs, _tinfos, _out_type):
```

Comment:
Any reason to use _ naming for inputs?

Reply:
Pylint complains these inputs are unused. Prefixing them with _ silences the warning and indicates to other people that the argument is unused.

Reply:
Ok, got it. Thanks for clarifying!
We can also use macro like below 
# pylint: disable=
def func():